featurebyte/session/databricks.py

```diff
@@ -86,6 +96,46 @@ def __init__(self, **data: Any) -> None:
             schema=self.featurebyte_schema,
         )
 
+    def _initialize_storage(self) -> None:
+        self.storage_spark_url = self.storage_spark_url.rstrip("/")
+        self._storage_base_path = self.storage_spark_url.lstrip("dbfs:")
+        # ensure google credentials not in environment variables to avoid conflict
+        os.environ.pop("GOOGLE_CREDENTIALS", None)
```

Comment:
Is this environment variable used elsewhere?

Reply:
not in the service but when set in dev env it causes the client to fail